### PR TITLE
Feature/add new issue landlord#3

### PR DESCRIPTION
**Pull request by Gubbsy**

### CODE REVIEW

File: src/app/views/landlord/issue-feed/issue-feed.page.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Component, Inject, OnInit } from "@angular/core";
 import LandlordFeedAPIService from "src/app/services/api/landlord/feed-api-service";
 import IIssueResponse from "src/app/models/response/landlord/issue-response.interface";
+import { Router } from '@angular/router';
```
Comment (tobysmith568) [inline src/app/views/landlord/issue-feed/issue-feed.page.ts]: Single quotes

File: src/app/views/landlord/new-issue/new-issue.module.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { FormsModule } from "@angular/forms";
+import { RouterModule } from "@angular/router";
+import { IonicModule } from "@ionic/angular";
+import { NewIssuePage } from "./new-issue.page";
+import LandlordCreateIssueAPIService from "src/app/services/api/landlord/create-issue-api-service";
+
+
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.module.ts]: Two line breaks

File: src/app/views/landlord/new-issue/new-issue.page.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <ion-title>Create New Issue</ion-title>
+    <ion-button slot="end" shape="round" size="medium" (click)="checkPressed()">
+      <ion-icon name="checkmark">
+      </ion-icon></ion-button>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <ion-item>
+      <ion-label class = "label">Priority</ion-label>
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.html]: Indentation

File: src/app/views/landlord/new-issue/new-issue.page.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-button slot="start" shape="round" size="medium" (click)="cancelPressed()">
+      <ion-icon name="close" ></ion-icon>
+    </ion-button>
+    <ion-title>Create New Issue</ion-title>
+    <ion-button slot="end" shape="round" size="medium" (click)="checkPressed()">
+      <ion-icon name="checkmark">
+      </ion-icon></ion-button>
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.html]: Any thoughts on these closing tags @Gubbsy  @tobybessant ? I'd drop the second down

File: src/app/views/landlord/new-issue/new-issue.page.spec.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.spec.ts]: Line break not needed

File: src/app/views/landlord/new-issue/new-issue.page.ts [hunk 6]
```diff
@@ -0,0 +1,105 @@
+import { Component, OnInit } from "@angular/core";
+import LandlordCreateIssueAPIService from "src/app/services/api/landlord/create-issue-api-service";
+import { Router } from "@angular/router";
+import { Location } from "@angular/common";
+import { ToastController, PickerController } from "@ionic/angular";
+import { PickerOptions } from '@ionic/core';
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.ts]: Single quotes

File: src/app/views/landlord/new-issue/new-issue.page.ts [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+    const toast = await this.toastController.create({
+      message: "All fields must be filled to submit an issue",
+      duration: 2000
+    });
+    toast.present();
+  }
+
+  async checkPressed(): Promise<void> {
+    if (this.isFormValid) {
+      //FIX: Payload always has priority 1
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.ts]: Fix?

**Review by tobysmith568 — CHANGES_REQUESTED** [review]

File: src/app/views/landlord/new-issue/new-issue.page.ts (hunk 7 above)
Comment (tobybessant) [inline src/app/views/landlord/new-issue/new-issue.page.ts]: Comment should be removed?

File: src/app/views/landlord/new-issue/new-issue.page.html (hunk 4 above)
Comment (tobybessant) [reply src/app/views/landlord/new-issue/new-issue.page.html]: @tobysmith568  Agreed

**Review by tobysmith568 — APPROVED** [review]

**Review by tobybessant — APPROVED** [review]

**Pull request merged**